editor/shader_create_dialog.cpp

```diff
@@ -163,8 +163,8 @@ void fragment() {
 }
 
 //void light() {
-	// Called for every pixel for every light affecting the material.
-	// Uncomment to replace the default light processing function with this one.
+	// // Called for every pixel for every light affecting the material.
+	// // Uncomment to replace the default light processing function with this one.
```

Comment:
Would this work instead? I think this would look clearer.
```suggestion
//	// Called for every pixel for every light affecting the material.
//	// Uncomment to replace the default light processing function with this one.
```

Reply:
Sure, that one uncomments fine with the shortcut, too.